server/integration/feature-pack/src/main/resources/modules/system/layers/base/org/hibernate/search/engine/main/module.xml

```diff
@@ -8,6 +8,7 @@
         <module name="javax.transaction.api" />
         <module name="org.hibernate.commons-annotations" />
         <module name="org.apache.lucene" export="true" />
+        <module name="org.apache.lucene.internal" export="true" />
```

Comment:
@anistor answering your question from the chat, the lucene-internal module was originally created as [part of ](https://github.com/hibernate/hibernate-search/commit/f9f547a5cdd877ab447a5bdcc391c5acec96ca5a) https://hibernate.atlassian.net/browse/HSEARCH-1849 and judging by the contents, the goal is to host older codecs and compatibility with older Lucene versions.

So maybe we could follow the wirings done in hibernate search that it'd work. Check [here](https://github.com/hibernate/hibernate-search/blob/master/modules/src/main/modules/search/engine/module.xml), [here](https://github.com/hibernate/hibernate-search/blob/master/modules/src/main/modules/lucene/module.xml) and [here](https://github.com/hibernate/hibernate-search/blob/master/modules/src/main/modules/lucene_internal/module.xml)

WRT ```wildfly-modules```, this not required since we simply explode hibernate search modules in the our zip. Probably we just need to test that the Uninverter works.